samples/subsys/mgmt/mcumgr/smp_svr/README.rst

```diff
@@ -287,80 +237,47 @@ For clarity, here is DTS label to slot to ``<image>`` translation table:
     | "image-3" |        |     3      |
     +-----------+--------+------------+
 
-.. note::
-
-   The ``-e`` option actually means "no erase", and is provided to the mcumgr
-   to prevent it from sending erase command to target, before updating image.
-   The options is always needed when ``-n`` is used for image selection,
-   as the erase command is hardcoded to erase slot-1 ("image-1"),
-   regardless of which slot is uploaded at the time.
-
 Upload the signed image
 =======================
 
-To upload the signed image, use the following command:
-
-.. code-block:: console
-
-   sudo mcumgr <connection string> image upload build/zephyr/zephyr.signed.bin
+To upload the signed image, refer to the documentation for your chosen tool, select the new
+firmware file to upload and begin the upload.
 
 .. note::
 
    At the beginning of the upload process, the target might start erasing
-   the image slot, taking several dozen seconds for some targets.  This might
-   cause an NMP timeout in the management protocol tool. Use the
-   ``-t <timeout-in-seconds`` option to increase the response timeout for the
-   ``mcumgr`` command line tool if this occurs.
+   the image slot, taking several dozen seconds for some targets.
 
 List the images
 ===============
 
-We can now obtain a list of images (slot-0 and slot-1) present in the remote
-target device by issuing the following command:
-
-.. code-block:: console
-
-   sudo mcumgr <connection string> image list
-
-This should print the status and hash values of each of the images present.
+We can now obtain a list of images (slot-0 and slot-1) present in the remote target device using
+the tool of your choice, which should print the status and hash values of each of the images
+present.
 
 Test the image
 ==============
 
-In order to instruct MCUboot to swap the images we need to test the image first,
-making sure it boots:
-
-.. code-block:: console
-
-   sudo mcumgr <connection string> image test <hash of slot-1 image>
-
-Now MCUBoot will swap the image on the next reset.
+In order to instruct MCUboot to swap the images we need to test the image first, making sure it
+boots, see the instructions in the tool of your choice. Upon reboot, MCUBoot will swap to the new
+image.
 
 .. note::
    There is not yet any way of getting the image hash without actually uploading the
-   image and getting the hash by using the ``image list`` command of :file:`mcumgr`.
+   image and getting the hash.
 
 Reset remotely
 ==============
 
-We can reset the device remotely to observe (use the console output) how
-MCUboot swaps the images:
-
-.. code-block:: console
-
-   sudo mcumgr <connection string> reset
-
-Upon reset MCUboot will swap slot-0 and slot-1.
+We can reset the device remotely to observe (use the console output) how MCUboot swaps the images,
+check the documentation in the tool of your choice. Upon reset MCUboot will swap slot-0 and
```

Comment:
non blocking but the mixed use of "you" vs. "we" across this page should probably be improved at some point

Reply:
Made some changes